### PR TITLE
Prepare docs for 2.14

### PR DESCRIPTION
**Pull request by wozniakjan**
Preparing docs for the upcoming 2.13 release with a scaffold for future 2.14

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

see also: https://github.com/kedacore/keda/issues/5275

### CODE REVIEW

**Comment by tomkerkhove** [comment]
Just FYI that I forgot to add https://github.com/kedacore/keda-docs/pull/1292 to 2.13 docs and need to backport to 2.14

**Comment by wozniakjan** [comment]
I will make sure to include this, thanks!

**Comment by wozniakjan** [comment]
will check if needs another rebase once the PRs mentioned in https://github.com/kedacore/keda/issues/5275#issuecomment-1894050761 merge

File: content/docs/2.13/scalers/memory.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ go_file = "cpu_memory_scaler"
 
 ### Prerequisites
 
-KEDA uses standard `cpu` and `memory` metrics from the Kubernetes Metrics Server, which is not installed by default on certain Kubernetes deployments such as EKS on AWS. Additionally, the `resources` section of the relevant Kubernetes Pods must include `requests` (at a minimum).
+KEDA uses standard `cpu` and `memory` metrics from the Kubernetes Metrics Server, which is not installed by default on certain Kubernetes deployments such as EKS on AWS. Additionally, the `resources` section of the relevant Kubernetes Pods must include at least one of `requests` or `limits`.
```
Comment (wozniakjan) [inline content/docs/2.13/scalers/memory.md:16]: this was added to CPU scaler but we missed it for the memory scaler docs, this just brings the two pages to parity matching the implementation

https://github.com/kedacore/keda-docs/pull/1296#discussion_r1455677499

Comment (wozniakjan) [inline content/docs/2.13/scalers/memory.md:16]: this was added to CPU scaler but we missed it for the memory scaler docs, this just brings the two pages to parity matching the implementation

https://github.com/kedacore/keda-docs/pull/1296#discussion_r1455677499

**Review by JorTurFer — APPROVED** [review]

**Review by tomkerkhove — APPROVED** [review]
LGTM, thanks!

**Review by tomkerkhove — APPROVED** [review]
LGTM, thanks!

**Pull request merged**